awx/ui_next/src/screens/Inventory/InventoryList/InventoryList.jsx

```diff
@@ -188,6 +191,17 @@ function InventoryList({ i18n }) {
           ]}
           toolbarSearchableKeys={searchableKeys}
           toolbarRelatedSearchableKeys={relatedSearchableKeys}
+          headerRow={
+            <HeaderRow defaultSortKey="name" qsConfig={QS_CONFIG}>
+              <HeaderCell sortKey="name">{i18n._(t`Name`)}</HeaderCell>
```

Comment:
This is a very nice, declarative interface!

Reply:
I agree